mantidproject/mslice
# Fix flake8 warnings and add flake8 to travis build

Fixes most flake8 issues with the code and adds the checker to the build.

**To test:**

Start mslice and perform some cuts/slices to see that everything works. Most functionality is covered by unit tests.
